mock/src/lib.rs

```diff
@@ -28,12 +31,25 @@ lazy_static! {
         address!("0x000000000000000000000000000000000cafe444"),
         address!("0x000000000000000000000000000000000cafe555"),
     ];
+    /// Mock wallets used to generate correctly signed and hashed Transactions.
+    pub static ref MOCK_WALLETS: Vec<LocalWallet> = {
+        let mut rng1 = ChaCha20Rng::seed_from_u64(0xcafe0u64);
+        let mut rng2 = ChaCha20Rng::seed_from_u64(0xcafe1u64);
+        let mut rng3 = ChaCha20Rng::seed_from_u64(0xcafe2u64);
```

Comment:
First, do we need 3 rngs to generate 3 wallets? Why not initiate just 1 rng and share with 3 txs?
Second, if this mock depends on an rng, should we redefine this as a factory function that takes rng as an argument?

Reply:
I agree on the first topic. 

As for the second one, this will probably be addressed in: https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/684